docs/release/releasing.md

```diff
@@ -1,12 +1,32 @@
 # Releasing
-1. Release and publish a new version of library:
-    * It can be [triggered in GitHub Actions].
+
+1. Release and publish a new version of the library:
+    * Perform the release using the [GitHub Actions CI workflow](#performing-a-release) by following the steps outlined in the "Performing a Release" section of this document.
     * If you want to debug it, you can run it locally as per [`gradle-release` plugin] docs.
 2. Update the repo to the new version (via pull request):
     * Cut off the `Unreleased` section in the [changelog] with the new version.
     * Update the version in the [readme] installation section.
     * Clean up internal `compatibleWithPreviousVersion` usages if they exist.
 
+## Performing a Release
+
+1. Go to the "Actions" tab in your GitHub repository.
+2. Click on the "CI" workflow.
+3. Click on "Run workflow" in the top right corner.
+4. Select the branch you want to release from the dropdown list.
+5. In the "Release?" input field, enter the type of release (Major/Minor/Patch/None). If you don't want to create a release, enter 'None'.
+6. Optionally, in the "Prerelease?" input field, enter the prerelease stage (Alpha/Beta/RC/None). If you don't want to create a prerelease, enter 'None'.
+7. Click on the "Run workflow" button.
+
+## Release Types
```

Comment:
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes

https://semver.org/